packages/superset-ui-chart/package.json

```diff
@@ -27,6 +27,16 @@
   },
   "dependencies": {
     "@superset-ui/core": "^0.3.0",
+    "prop-types": "^15.6.2",
+    "react": "^16.4.1",
```

Comment:
could think about extending this if there are no known compatibility issues with e.g., all of 16 and / or 15

Reply:
Revised.